src/main/java/com/gravity9/jsonpatch/diff/JsonDiff.java

```diff
@@ -96,6 +102,53 @@ public static JsonPatch asJsonPatch(final JsonNode source,
 		return processor.getPatch();
 	}
 
+	/**
+	 * Generate a JSON patch for transforming the source node into the target
+	 * node ignoring given fields
+	 *
+	 * @param source the node to be patched
+	 * @param target the expected result after applying the patch
+	 * @param fieldsToIgnore list of JsonPath or JsonPointer paths which should be ignored when generating diff. Non-existing fields are ignored.
+	 * @return the patch as a {@link JsonPatch}
+	 * @since 1.9
+	 */
+	public static JsonPatch asJsonPatchIgnoringFields(final JsonNode source,
+										final JsonNode target, final List<String> fieldsToIgnore) {
+		BUNDLE.checkNotNull(source, "common.nullArgument");
+		BUNDLE.checkNotNull(target, "common.nullArgument");
+		final List<JsonPatchOperation> ignoredFieldsRemoveOperations = getJsonPatchRemoveOperationsForIgnoredFields(fieldsToIgnore);
+
+		JsonNode sourceWithoutIgnoredFields = removeIgnoredFields(source, ignoredFieldsRemoveOperations);
+		JsonNode targetWithoutIgnoredFields = removeIgnoredFields(target, ignoredFieldsRemoveOperations);
+
+		final Map<JsonPointer, JsonNode> unchanged
+				= getUnchangedValues(sourceWithoutIgnoredFields, targetWithoutIgnoredFields);
+		final DiffProcessor processor = new DiffProcessor(unchanged);
+
+		generateDiffs(processor, JsonPointer.empty(), sourceWithoutIgnoredFields, targetWithoutIgnoredFields);
+		return processor.getPatch();
+	}
+
+	private static JsonNode removeIgnoredFields(JsonNode node, List<JsonPatchOperation> ignoredFieldsRemoveOperations) {
+		JsonNode nodeWithoutIgnoredFields = node;
+		for (JsonPatchOperation operation : ignoredFieldsRemoveOperations) {
+			nodeWithoutIgnoredFields = removeIgnoredFieldOrIgnore(nodeWithoutIgnoredFields, operation);
+		}
+		return nodeWithoutIgnoredFields;
+	}
+
+	private static JsonNode removeIgnoredFieldOrIgnore(JsonNode nodeWithoutIgnoredFields, JsonPatchOperation operation) {
+		try {
+			List<JsonPatchOperation> operationsList = new ArrayList<>();
+			operationsList.add(operation);
+			return new JsonPatch(operationsList).apply(nodeWithoutIgnoredFields);
+		} catch (JsonPatchException e) {
+			// If remove for specific path failed, it means that node does not contain specific field which should be ignored.
+			// See more `empty patch if object does not contain ignored field` in diff.json file.
+			return nodeWithoutIgnoredFields;
```

Comment:
As discussed on the call, please only catch the Exception that relates to missing node (path parse exceptions should be rethrown for example)

Reply:
done.